R/tokens.R

```diff
@@ -341,17 +351,20 @@ tokens.tokens <-  function(x,
         x <- tokens_split(x, "\\p{Pd}", valuetype = "regex", remove_separator = FALSE)
     }
     
+    if (remove_separators)
+        x <- tokens_remove(x, "^[\\p{Z}\\p{C}]$", valuetype = "regex")
```

Comment:
you're sure this is faster than putting it into the removals_regex and calling `tokens_remove()` just once?

Reply:
It is faster to do it in one-go with `removals_regex` for sure, but I need to add a `tokens_remove()` to avoid paddings for separators when `padding = TRUE`.

Reply:
Right - actually this last line should not remove the pad.
```r
> as.tokens(list(d1 = c("one", " ", "x", "two"))) %>%
+     tokens_remove("x", padding = TRUE) %>%
+     tokens(remove_separators = TRUE)
Tokens consisting of 1 document.
d1 :
[1] "one" "two"
```
but if we could fix the regex to remove spaces but not pads, we could do it in one expression.